build/versionize-docs.sh

```diff
@@ -72,4 +72,5 @@ done
 $SED -ri -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|" pkg/api/v[0-9]*/types.go
 
 ${KUBE_ROOT}/hack/update-generated-docs.sh
-${KUBE_ROOT}/hack/update-swagger-spec.sh
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
+
```

Comment:
@caesarxuchao I think that @nikhiljindal 's comment means this is unnecessary.  If that's true, can I have an LGTM?

Thanks!
--brendan